LayoutTests/inspector/canvas/resources/recording-2d.js

```diff
@@ -84,6 +88,8 @@ function performActions() {
             ctx.clearRect(1, 2, 3, 4);
         },
         () => {
+            if (ctx instanceof OffscreenCanvasRenderingContext2D)
+                return;
```

Comment:
NIT: instead of early returning you could use optional chaining to only call the method if it exists
```
ctx.clearShadow?.();
```

Reply:
done.